src/app/component/MissionCard.jsx

```diff
@@ -53,9 +66,10 @@ const MissionCardContent = ({ classes, contentItems }) => (
  * @component
  */
 const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest }) => {
-  const title = mission.title;
+  const title = mission?.title || "No title supplied.";
```

Comment:
Looks like the title came out of the CardHeader / Card but do we still want it in the View? aka this title property?